pkg/crc/crc_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package crc
+
+import (
+	"hash/crc32"
+	"reflect"
+	"testing"
+)
+
+// TestHash32 tests that Hash32 provided by this package behaves exactly
+// the same as the standard one.
```

Comment:
we are actually testing the hash can take an initial crc and will generate the exact same output in the following calls.